keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/Health.kt

```diff
@@ -27,8 +27,21 @@ data class Health(
   val warmup: Duration = Duration.ofSeconds(600),
   val healthCheckType: HealthCheckType = HealthCheckType.EC2,
   val enabledMetrics: Set<Metric> = emptySet(),
-  val terminationPolicies: Set<TerminationPolicy> = setOf(TerminationPolicy.OldestInstance)
+  val terminationPolicies: Set<TerminationPolicy> = setOf(TerminationPolicy.Default)
```

Comment:
I changed this because `Default` is what is the default value in the UI.

Reply:
Please revert this, changing the spinnaker default ec2 termination policy to `OldestInstance` was a request from our SRE org that is also reflected in pipelines created by newt / project generators. It's a miss that deck hasn't been updated.

Reply:
Probably worth adding a code comment too! I didn't know/forgot that this was a request.

Reply:
Ah! Thanks for the heads-up! So are we going to change the default in the UI as well?
![image](https://user-images.githubusercontent.com/1323478/68351513-8eeb7700-00b8-11ea-871f-f371794ddc15.png)

Reply:
I hope so, feel like touching deck? :)

Reply:
Keep in mind that managed delivery in some ways is a vehicle to free us from past decisions. Whenever you see keel diverge from older norms, it certainly may be in error but it's just as likely intended to make something better.

Reply:
I'm gonna get this updated in the UI if possible, **but** — what do people think about auto-converting `Default` to `OldestInstance` when we see it in an existing server group we're exporting? Every pipeline created through the UI up until this moment will have been setup with `Default`, so anybody we have export their cluster will end up with `Default` in their YAML config and won't get any benefit whatsoever from us changing the default at the resource level.

Reply:
@erikmunson My vote goes against doing that as it would change what is currently expected behavior by the app teams of their infrastructure, and under their radar. For these cases, perhaps we could think about education campaigns or informational/warning messages that are integrated into the MD UI to let users know there's a better default, so they can make the change when they're ready, or even opt-in to an MD "auto-resolve"? Something like a "compliance view" of sorts? I have at least one tool in mind that I could show you that implements that concept.

Reply:
@erikmunson Based on this thread and offline discussion, I added a separate issue to handle this: https://github.com/spinnaker/keel/issues/605